recipes/aws-sdk-cpp/all/conanfile.py

```diff
@@ -0,0 +1,250 @@
+            "cur",
+            "datapipeline",
+            "dax",
+            "devicefarm",
+            "directconnect",
+            "discovery",
+            "dlm",
+            "dms",
+            "docdb",
+            "ds",
+            "dynamodb",
+            "dynamodbstreams",
+            "ec2",
+            "ecr",
+            "ecs",
+            "eks",
+            "elasticache",
+            "elasticbeanstalk",
+            "elasticfilesystem",
+            "elasticloadbalancing",
+            "elasticloadbalancingv2",
+            "elasticmapreduce",
+            "elastictranscoder",
+            "email",
+            "es",
+            "events",
+            "firehose",
+            "fms",
+            "fsx",
+            "gamelift",
+            "glacier",
+            "globalaccelerator",
+            "glue",
+            "greengrass",
+            "guardduty",
+            "health",
+            "iam",
+            "identity_management",
+            "importexport",
+            "inspector",
+            "iot_data",
+            "iot_jobs_data",
+            "iot",
+            "kafka",
+            "kinesis",
+            "kinesisanalytics",
+            "kinesisvideo",
+            "kms",
+            "lambda",
+            "lex",
+            "lightsail",
+            "logs",
+            "machinelearnings",
+            "macie",
+            "marketplace_entitlement",
+            "marketplacecommerceanalytics",
+            "mediaconvert",
+            "medialive",
+            "mediapackage",
+            "mediastore",
+            "mediatailor",
+            "meteringmarketplace",
+            "mobileanalytics",
+            "monitoring",
+            "mq",
+            "mturk_requester",
+            "neptune",
+            "opsworks",
+            "opsworkscm",
+            "organizations",
+            "pinpoint",
+            "polly",
+            "pricing",
+            "queues",
+            "quicksight",
+            "ram",
+            "rds",
+            "redshift",
+            "recognition",
+            "resource_groups",
+            "robomaker"
+            "route53",
+            "route53domains",
+            "s3",
+            "sagemaker",
+            "sdb",
+            "serverlessrepo"
+            "servicecatalog",
+            "servicediscovery",
+            "shield",
+            "signer",
+            "sms",
+            "snowball",
+            "sns",
+            "sqs",
+            "ssm",
+            "states",
+            "storagegateway",
+            "sts",
+            "support",
+            "swf",
+            "text_to_speech",
+            "texttract",
+            "transcribe",
+            "transfer",
+            "translate",
+            "waf",
+            "workdocs",
+            "worklink",
+            "workmail",
+            "workspaces",
+            "xray"
+           )
+    options = {
+            **{ x: [True, False] for x in sdks },
+            **{
```

Comment:
does this work with python 2? we still need to support that...

Reply:
Conan dropped python 2 support since 1.25.0.

Reply:
This is not true. Python 2 is supported through the 1.x series. Only Conan 2 will increase the minimum to 3.6, see https://github.com/conan-io/tribe/blob/main/design/003-codebase-python.md

It is true, that the Python 2 tag [got removed from meta data for PyPi for some reason,](https://github.com/conan-io/conan/pull/6787) but support is unchanged

Reply:
https://docs.conan.io/en/latest/installation.html#python-2-deprecation-notice

> Starting in Conan 1.22, Python 2 is no longer supported and Conan will stop working with it in the following releases.
>
> Conan 2.x won’t support Python 2.

The documentation is very confusing... is it support or not?

Reply:
Indeed, now I'm confused too 

@jgsogo and @danimtb could you please shine some light on the Python 2 support status?

Reply:
The previous paragraph should clarify a bit: 

> All features of Conan until version 1.6 are fully supported in both Python 2 and Python 3. However, new features in upcoming Conan releases that are only available in Python 3 or more easily available in Python 3 will be implemented and tested only in Python 3, and versions of Conan using Python 2 will not have access to that feature. This will be clearly described in code and documentation.

We are still running all our test suite (I'm talking about Conan client) with Python2, so far we haven't found any blockers (there is always a workaround), and our requirements are fixed and/or they still support Python 2. But if it were the case, we would drop Python2 from our testing environment.

TL;DR; We will do our best to keep supporting Python2 in Conan v1.x.  About recipes in ConanCenter: here we are only using Python3, I would hope everything work in Python2 as well, but IMHO it is not a blocker (if the CI is green, the PR is _mergeable_).